forest/jasmine/traj2stats.py

```diff
@@ -732,182 +1379,126 @@ def gps_summaries(
-                for cname in list(summary_stats_df.columns)[3:]
-            ]
-            + [
-                f"{cname}_nighttime"
-                for cname in list(summary_stats_df.columns)[3:]
-            ]
-        )
-        summary_stats_df2 = summary_stats_df2.drop(
-            [
-                "obs_day_daytime",
-                "obs_night_daytime",
-                "obs_day_nighttime",
-                "obs_night_nighttime",
-            ],
-            axis=1,
-        )
-        summary_stats_df2.insert(
-            3,
-            "obs_duration",
-            summary_stats_df2["obs_duration_daytime"]
-            + summary_stats_df2["obs_duration_nighttime"],
-        )
-    else:
-        summary_stats_df2 = summary_stats_df
+    summary_stats_df2, log_tags = format_summary_stats(
+        summary_stats, log_tags, frequency, parameters, places_of_interest
+    )
 
     return summary_stats_df2, log_tags
 
 
+def split_day_night_cols(summary_stats_df: pd.DataFrame) -> pd.DataFrame:
+    """This function splits the summary statistics dataframe
+    into daytime and nighttime columns.
+
+    Args:
+        summary_stats_df: pandas dataframe with summary statistics
+    Returns:
+        summary_stats_df2: pandas dataframe with summary statistics
+            split into daytime and nighttime columns
+    """
+
+    summary_stats_df_daytime = summary_stats_df[::2].reset_index(drop=True)
+    summary_stats_df_nighttime = summary_stats_df[1::2].reset_index(drop=True)
+
+    summary_stats_df2 = pd.concat(
+        [
+            summary_stats_df_daytime,
+            summary_stats_df_nighttime.iloc[:, 3:],
+        ],
+        axis=1,
+    )
+    summary_stats_df2.columns = (
+        list(summary_stats_df.columns)[:3]
+        + [
+            f"{cname}_daytime"
+            for cname in list(summary_stats_df.columns)[3:]
+        ]
+        + [
+            f"{cname}_nighttime"
+            for cname in list(summary_stats_df.columns)[3:]
+        ]
+    )
+    summary_stats_df2 = summary_stats_df2.drop(
+        [
+            "obs_day_daytime",
+            "obs_night_daytime",
+            "obs_day_nighttime",
+            "obs_night_nighttime",
+        ],
+        axis=1,
+    )
+    summary_stats_df2.insert(
+        3,
+        "obs_duration",
+        summary_stats_df2["obs_duration_daytime"]
+        + summary_stats_df2["obs_duration_nighttime"],
+    )
+
+    return summary_stats_df2
+
+
+def get_time_range(
+    traj: np.ndarray, time_reset_indices: list,
+    tz_str: str, offset_seconds: int = 0,
+) -> Tuple[int, int]:
+    """Computes the starting and ending time stamps
+     based on given trajectory and indices.
+
+    Args:
+        traj: numpy array of trajectory
+        time_reset_indices: list of indices to reset time
+        offset_seconds: int, offset in seconds
+        tz_str: str, timezone
+    Returns:
+        start_stamp: int, starting time stamp
+        end_stamp: int, ending time stamp
```

Comment:
"Do not imitate older ‘NumPy style’ ([example](https://numpy.org/doc/1.24/reference/generated/numpy.linalg.qr.html)), which frequently documented a tuple return value as if it were multiple return values with individual names (never mentioning the tuple). Instead, describe such a return value as: “Returns: A tuple (mat_a, mat_b), where mat_a is …, and …”.
https://google.github.io/styleguide/pyguide.html#doc-function-returns